airflow/providers/amazon/aws/sensors/sqs.py

```diff
@@ -136,15 +135,27 @@ def poke(self, context: 'Context'):
             messages = self.filter_messages(messages)
             num_messages = len(messages)
             self.log.info("There are %d messages left after filtering", num_messages)
+        return messages
+
+    def poke(self, context: 'Context'):
+        """
+        Check for message on subscribed queue and write to xcom the message with key ``messages``
 
-        if not num_messages:
+        :param context: the context object
+        :return: ``True`` if message is available or ``False``
+        """
+        sqs_conn = self.get_hook().get_conn()
```

Comment:
I know this is how it was done elsewhere in this file, and someone may correct me if I'm wrong, but I think we're supposed to be moving towards the following way of accessing the connection.  If that's right, then could you make that change while you are in here?
```suggestion
        sqs_conn = self.get_hook().conn
```

Reply:
Indeed, I did not pay attention to that